kedro-telemetry/kedro_telemetry/plugin.py

```diff
@@ -41,6 +43,7 @@
     "BUILDKITE",  # https://buildkite.com/docs/pipelines/environment-variables
 }
 TIMESTAMP_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
+CONFIG_FILENAME = "telemetry.conf"
```

Comment:
> is it expected to be commited into the repo?

You mean this file?  It will be stored in the user's configuration directory, not in project directory, so looks like it will not be possible to commit it.